orderer/common/server/main.go

```diff
@@ -372,13 +374,9 @@ func initializeLogging() {
 
 // Start the profiling service if enabled.
 func initializeProfilingService(conf *localconfig.TopLevel) {
-	if conf.General.Profile.Enabled {
-		go func() {
```

Comment:
the UT `TestInitializeProfilingService` needs to be altered to accommodate this change - this becomes a blocking method now

Reply:
I believe you will need to modify https://github.com/hyperledger/fabric/blob/master/orderer/common/server/main_test.go#L76 to use a goroutine

Reply:
Nice catch!